test/routes/routes.test.js

```diff
@@ -0,0 +1,95 @@
+const chai = require('chai');
+const request = require('request-promise-native');
+const app = require('../../src/app');
+const getAllRoutes = require('../services/helpers/getAllRoutes');
+const { whitelist, ignoreList } = require('./whitelist');
+const { API_HOST } = require('../../config/globals');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['API_HOST' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5374867)